sphinx_scylladb_theme/header.html

```diff
@@ -78,6 +78,7 @@
         </div>
     </div>
     <div class="header-search-box">
```

Comment:
I can't see the logo in dark mode:

![image](https://github.com/user-attachments/assets/f6a13724-272c-448f-94ae-cc138fccd193)